frontend/public/components/graphs/prometheus-rules-hook.ts

```diff
@@ -0,0 +1,18 @@
+import { useURLPoll } from '../utils/url-poll-hook';
+import { getPrometheusURL, PrometheusEndpoint } from './helpers';
+import { PrometheusRulesResponse } from '../monitoring/types';
+
+export const usePrometheusRulesPoll = ({ delay, namespace }: PrometheusPollProps) => {
+  const url = getPrometheusURL({
+    endpoint: PrometheusEndpoint.RULES,
+    namespace,
+    query: `namespace=${namespace}`,
+  });
```

Comment:
Here I have a just question, is this change the expected behaviour? I double checked the API calls to the backend, and the query parameter is new here:

Old: `/api/prometheus-tenancy/api/v1/rules?namespace=christoph-test`
New: `/api/prometheus-tenancy/api/v1/rules?namespace=christoph-test&query=namespace%3Dchristoph-test`

Is this "okay" or unnecessary?

Reply:
Right, unnecessary `query` prop in `metricUtils.ts`